frontend/src/components/object/ObjectUpload.vue

```diff
@@ -49,8 +49,8 @@ const onUpload = async (event: any) => {
           // Infinite timeout for big files upload to avoid timeout error
           await objectStore.createObject(file, bucketId, { timeout: 0 });
           successfulFiles.value.push(file);
-        } catch (error) {
-          toast.add({ severity: 'error', summary: 'Error', detail: `Failed to upload file ${file.name}`, life: 3000 });
+        } catch (error: any) {
+          toast.error('Uploading file', error);
```

Comment:
Style: Would be ideal to preserve more precise wording when an error occurs. Does passing the raw error still let the user know which file failed?